runtime/compiler/runtime/RelocationRecord.cpp

```diff
@@ -2973,20 +3009,8 @@ TR_RelocationRecordInlinedVirtualMethod::name()
    return "TR_InlinedVirtualMethod";
    }
 
-void
-TR_RelocationRecordInlinedVirtualMethod::print(TR_RelocationRuntime *reloRuntime)
-   {
-   Base::print(reloRuntime);
-   }
-
-void
-TR_RelocationRecordInlinedVirtualMethod::preparePrivateData(TR_RelocationRuntime *reloRuntime, TR_RelocationTarget *reloTarget)
-   {
-   Base::preparePrivateData(reloRuntime, reloTarget);
-   }
-
 TR_OpaqueMethodBlock *
-TR_RelocationRecordInlinedVirtualMethod::getMethodFromCP(TR_RelocationRuntime *reloRuntime, void *void_cp, int32_t cpIndex)
+TR_RelocationRecordInlinedVirtualMethod::getMethodFromCP(TR_RelocationRuntime *reloRuntime, void *void_cp, int32_t cpIndex, TR_OpaqueMethodBlock *callerMethod)
```

Comment:
We were never calling `TR_RelocationRecordInlinedVirtualMethod::getMethodFromCP` when the relo kind was `TR_InlinedVirtualMethod` because this virtual method was missing the `TR_OpaqueMethodBlock *callerMethod` param; `TR_RelocationRecordInlinedMethod::getMethodFromCP` ended up getting called, which would just cause a load failure...